applications/launchpad/README.md

```diff
@@ -0,0 +1,126 @@
+
+
+
+## Viewing logs and configuration files
+
+You can use the bundled Grafana environment that is packaged with launchpad to view log files. Or you can use your 
+favorite text editor instead.
+
+Logs and configuration files are stored in 
+* MacOs: `~/Library/Caches/tari/tmp/dibbler/{app}/log`,
+* Linux: `~/.cache/tari/tmp/dibbler/{app}/log`,
+* Windows: `???`
+
+You can edit the log configuration, `dibbler/config/log4rs.yml` to change the log level, output etc. Changes are 
+picked up on the fly and take effect within 30s.
+
+##  Miscellaneous notes
+* 
+* The blockchain data is stores in docker volumes, and not on the host machine directly. This is due to crippling performance
+limitations one suffers when mounting host file system from Windows or MacOS into docker containers.
+This isn't a big drawback, since you seldom want or need to access the raw blockchain database files anyway. Are they're
+[still accessible](#accessing-blockchain-data). But **ensure that you reserve enough space to store the Tari, and optionally,
+Monero blockchains inside the Docker VM**.
+
+### Accessing blockchain data
+
+The blockchain data is stored in a docker volume for performance reasons. If you need to back up or access the LMDB
+a blockchain data, you can use something like this to extract it to the host filesystem:
+
+`docker run --rm -v $(pwd):/backup -v blockchain:/blockchain ubuntu tar czvf /backup/backup.tar.gz /blockchain`
+
+
+## Layout
+
+      +-----------------------+
+      |                       |
++---->|    Console Wallet     +------------------+
+|     |                       |                  |
+|     +----------+------------+                  |
+|                |                               |
+|                | gRPC                          |
+|                |                               |
+|                |                               |
+|     +----------v------------+           +------v-----+
+|     |                       |  Socks5   |            |
+|     |       Base Node       +---------->|     Tor    |----> Network
+|     |                       |           |            |
+|     +----------^------------+           +------------+
+|                |
+|                |
+|                |
+|     +----------+------------+
+|     |                       |
++-----+      SHA3-Miner       |
+|     |                       |
+|     +-----------------------+
+|
+|
+|
+|     +-----------------------+
+|     |                       |
++-----+        XMRRig etc     |
+      |                       |
+      +-----------------------+
+
+## Building custom docker images
+
+The docker images for the base node, wallet etc. are designed to handle the broadest set of chipsets and 
+architectures. For this reason, they not be optimal for _your_ system. You can build custom images for launchpad 
+using the `build_images.sh` script in this folder.
+
+You can also set the following environment variables that are passed to the rust compiler:
```

Comment:
Need to review the info on envs and notes for them.

There are three samples current. And you can't really build/target none native architecture/platform.